corehq/motech/dhis2/templates/dhis2/edit_config.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'hqwebapp/two_column.html' %}
+{% load crispy_forms_tags %}
+{% load hq_shared_tags %}
+{# todo: this was just copied from userreports/userreports_base.html #}
+{% block title %}Edit Config : DHIS2 :: {% endblock %}
+{% block js %}{{ block.super }}
+    <script src="{% static 'codemirror/lib/codemirror.js' %}"></script>
+    <script src="{% static 'codemirror/mode/javascript/javascript.js' %}"></script>
+    <script src="{% static 'codemirror/addon/fold/foldcode.js' %}"></script>
+    <script src="{% static 'codemirror/addon/fold/foldgutter.js' %}"></script>
+    <script src="{% static 'codemirror/addon/fold/brace-fold.js' %}"></script>
+    <script src="{% static 'userreports/js/base.js' %}"></script>
+{% endblock %}
+{% block stylesheets %}{{ block.super }}
+    <link rel="stylesheet" href="{% static 'codemirror/lib/codemirror.css' %}" />
+    <link rel="stylesheet" href="{% static 'codemirror/addon/fold/foldgutter.css' %}"/>
+    <style>
+        .CodeMirror {
```

Comment:
We recently (like only a few days ago) merged code transitioning from CodeMirror to Ace. Fortunately [the code to use Ace][1] is shorter than the code here for CodeMirror. (You will probably need to merge master into your branch.)


  [1]: https://github.com/dimagi/commcare-hq/blob/ee4cff6a6b6ecd892a26c54d433c1005664df5ba/corehq/motech/openmrs/templates/openmrs/edit_config.html#L6